src/elm/Main.elm

```diff
@@ -82,9 +82,17 @@ init flags =
         section4draggableContent =
             ( Data.Section4, content.images |> Data.filterBySection Data.Section4 |> List.map Pile.Image )
 
+        section8draggableContent : ( Data.SectionId, List Pile.Data )
+        section8draggableContent =
+            ( Data.Section8, content.images |> Data.filterBySection Data.Section8 |> List.map Pile.Image )
+
         section10draggableContent : ( Data.SectionId, List Pile.Data )
         section10draggableContent =
             ( Data.Section10, content.posts |> Data.filterBySection Data.Section10 |> List.map Pile.Post )
+
+        section16draggableContent : ( Data.SectionId, List Pile.Data )
```

Comment:
Might be nice to pull this into a function in `Pile` rather than repeat in this `let .. in` but fine to leave it too.

You should be able to figure which of Pile.Image or Pile.Post to use from the content - or add that as a param if necessary.

```
dragablePile : Data.SectionId -> List dragableItem -> (Data.SectionId, List Pile.Data)
```

Reply:
I had a little look into this and couldn't figure out how to match on content type of a list when it's a type alias. I'm going to merge it in now and keep it as a brain teaser for the future.

Reply:
Sounds good.